concrete/composer.json

```diff
@@ -97,7 +97,7 @@
     "indigophp/hash-compat": "^1.1",
     "phpseclib/phpseclib": "^3.0",
     "symfony/psr-http-message-bridge": "^2.0",
-    "guzzlehttp/guzzle": "^7.4",
+    "guzzlehttp/guzzle": "7.5.3.",
```

Comment:
What about `"^7.5.3"` instead of `"7.5.3."` ?

That is, "at least 7.5.3 but still compatible to it" (and without the final dot).